front/src/components/AdminPanel/Panel/InsertQualificationScores/InsertQualificationScores.js

```diff
@@ -0,0 +1,652 @@
+        }
+        updateGetAllCurrentScores();
+    };
+
+    let isValidX_10Input = (event) => {
+        const id = event.target.id;
+        let input = event.target.value;
+        blockAndCleanArrowsInputs(id.split("-")[1]);
+        if (input.length > 1) document.getElementById(id).value = "";
+        if (
+            !/\D/.test(input) &
+            (parseInt(input) >= 0) &
+            (parseInt(input) <= 6)
+        ) {
+        } else {
+            document.getElementById(id).value = "";
+        }
+    };
+
+    function arrowInputs(competitorId) {
+        let inputs_1_3 = (
+            <div>
+                <Row className="alignToCenter">
+                    {[1, 2, 3].map((i) => (
+                        <input
+                            type="text"
+                            className="smallInput "
+                            key={"i" + i + "-" + competitorId}
+                            id={"i" + i + "-" + competitorId}
+                            onChange={isValidScoreInput}
+                        />
+                    ))}
+                </Row>
+            </div>
+        );
+        let inputs_4_6 = (
+            <div>
+                <Row className="alignToCenter">
+                    {[4, 5, 6].map((i) => (
+                        <input
+                            type="text"
+                            className="smallInput"
+                            key={"i" + i + "-" + competitorId}
+                            id={"i" + i + "-" + competitorId}
+                            onChange={isValidScoreInput}
+                        />
+                    ))}
+                </Row>
+            </div>
+        );
+
+        if (seriesType === 3) return <div key={competitorId}>{inputs_1_3}</div>;
+        else if (seriesType === 6)
+            return (
+                <div key={competitorId}>
+                    {inputs_1_3}
+                    {inputs_4_6}
+                </div>
+            );
+    }
+
+    let checkSendedScores = () => {
+        let inputsCheck = true;
+        for (var i = 0; i < currentParticipantsScores.length; i++) {
+            if (
+                !document.getElementById(
+                    "sum-" + currentParticipantsScores[i]._id
+                ).disabled &
+                !document.getElementById(
+                    "i1-" + currentParticipantsScores[i]._id
+                ).disabled
+            ) {
+                var t = 0;
+                for (var t = 0; t < seriesType; t++) {
+                    if (currentParticipantsScores[i].arrows[t] === "") t++;
+                }
+                if (t < seriesType) inputsCheck = false;
+                if (
+                    currentParticipantsScores[i].score === "" ||
+                    currentParticipantsScores[i].Xs === "" ||
+                    currentParticipantsScores[i].tens === ""
+                ) {
+                    inputsCheck = false;
+                }
```

Comment:
You used 't' as variable name twice, making code very hard to read. I am not even sure if *you* know how it works :)
Please, refactor it

Reply:
small bug,
changed checker variable name for more self-explaining